src/patterns/addresses/index.md.njk

```diff
@@ -84,7 +84,7 @@ If you use multiple text inputs, you should:
 
 Make sure there are enough text inputs to accommodate longer addresses if you know your users will need them. For example, allow users to include a company name or flat&nbsp;number.
 
-Royal Mail does not need a county as long as the town and postcode are included. You should include an optional county text input so that people who do not know their postcode can give a valid address.
+Avoid including county unless you have good evidence of a need for it. It is not part of a correct address according to Royal Mail and is not used to deliver mail.
```

Comment:
Looking good, thanks @terrysimpson99. How's this look to front-load the instruction a bit? 

```suggestion
Do not ask for the county unless you have good evidence of a need for it. It's not part of a correct address, according to Royal Mail, and it's not used to deliver mail.
```

Wondering if we even need "unless you have good evidence of a need for it"? I suppose it depends if they're used to deliver other services (such as by local authorities).

Smaller thought that 'county' might look like 'country' at a quick glance, but not sure if it'd be worth the space to use examples to clarify that:

```suggestion
Do not ask for the county (such as Leicestershire or Cumbria) unless you have good evidence of a need for it. It's not part of a correct address, according to Royal Mail, and it's not used to deliver mail.
```

Do you have any thoughts to offer @EoinShaughnessy?

Reply:
@calvin-lau-sig7 

- I think you've made the guidance easier to use by replacing the fuzzy '_Avoid_' with the unambiguous '_Do not_'.
- I agree 'unless you have good evidence of a need for it' is probably redundant. That applies to all guidance anyway. Happy for it to be removed.
- With regard to 'county' being mistaken for 'country': instead of county examples, how about replacing '_correct address_' with '_correct UK address_'. Just a thought.

Reply:
The three versions posted by both of you ( @calvin-lau-sig7 and @EoinShaughnessy ) seem workable to me.

Reply:
Thanks both! The paragraph still feels pretty concise to me with everything that's in there so far (including 'correct UK address'), so I don't mind if we commit with that?

```suggestion
Do not ask for the county (such as Berkshire or Cumbria) unless you have good evidence of a need for it. It's not part of a correct UK address, according to Royal Mail, and it's not used to deliver mail.
```